util/util.go

```diff
@@ -38,6 +38,10 @@ func commonFlags() []cli.Flag {
 			Name:  "table",
 			Usage: "Return output in tabular format. This is the default output format.",
 		},
+		cli.BoolFlag{
+			Name:  "csv",
+			Usage: "Return output in csv format.",
+		},
```

Comment:
I prefer `output` as an explicit flag. it's clear that you're talking about output format (for example, you might later be able to supply an input format for advanced calls).

It also prevents us from having to worry about argument clutter over time as we might change/add/remove output options.

```
rack servers instance list --output json
```